kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx

```diff
@@ -137,9 +137,13 @@ const Details: React.FC = () => {
       <Table isFullwidth>
         <thead>
           <tr>
-            <TableHeaderCell> </TableHeaderCell>
-            <TableHeaderCell title="Topic" />
-            <TableHeaderCell title="Messages behind" />
+            <S.TableHeaderConsumerCell size="small" title="Topic">
```

Comment:
`title` props are not being used , we should remove them.

Reply:
@Mgrdich removed title props